source/features/hide-inactive-deployments.js

```diff
@@ -0,0 +1,22 @@
+import select from 'select-dom';
+import observeEl from '../libs/simplified-element-observer';
+
+export default function () {
+	observeEl('.js-discussion', () => {
```

Comment:
Deployments only appear in PRs, right? Then before `observeEl` I think we need this:

```js
if (!pageDetect.isPR()) {
	return;
}
```

Reply:
I thought this is done in `content.js`, but I added it anyways.

Reply:
We’re trying to move away from that, so that the feature is self-contained